bin/bootstrap.sh

```diff
@@ -46,7 +46,7 @@ done
 for f in '.gitignore' '.prettierrc.yml' 'README.md'; do
   cp $PWD/.values/$f $ENV_DIR/
 done
-if [ -z "$(ls -A $ENV_DIR/env)" ]; then
+if [ ! -d "$ENV_DIR/env" ]; then
```

Comment:
I originally created the check but there's `-d` for that so `ls -A` will fail in edge cases (e.g. $ENV_DIR not set).